### ISSUE TITLE
Key Bind

### ISSUE THREAD

**Issue by siesoahoratellamo** [DerpleMQ2/buttonmaster#13]
It's possibleIt is possible to create a keybing system for some button, 1 2 3 4 5... up to 0 would be ideal to completely replace normal boptons

**Comment by burdsjm** [comment]
Yeah, that would be game-changing for me.  Derple, you can do it!

**Comment by DerpleMQ2** [comment]
You can bind keys - sort of - right now there is a bug with ISB that prevents MQ for detecting key presses correctly but you can add ingame hotkey bound to keys and then use the / commands to execute buttonmaster commands.

**Comment by burdsjm** [comment]
I don't use ISB.  Can you give me an example?

**Comment by DerpleMQ2** [comment]
/timed 50 /echo this happened some time later
/echo this happened first
